src/UI/Component/Panel/Listing/Appointement.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Interface Appointment
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Appointment extends \ILIAS\UI\Component\Component {
+	/**
+	 * Gets the title of the appointment listing
+	 *
+	 * @return string
+	 */
+	public function getTitle();
+
+	/**
+	 * Get the link pointing back to target for getting more
+	 * items (e.g. if selected a larger count of days to be displayed)
+	 *
+	 * @return string
+	 */
+	public function getAction();
```

Comment:
@alex40724 Note, this is the action of the complete list, not of a single item. Action is probably a bad name (as indicated by your question, it is far too generic and does not tell anything about it's true intend). Idea here is to have an option to set an async link back to get more items. This can be used to perform async filtering or similar (only implicitly visible on the mockups). Also note the JF comment for "Show More". For such behaviour, an async url to some point where the consumer of this list can push more items to the list is definitely needed.